en/sql/function/analysis_fn.rst

```diff
@@ -2047,3 +2047,58 @@ The following example shows how to output the score and sample variance of all s
                 3  'Lee'                    9.300000000000000e+01     5.435000000000000e+02
                 3  'Sara'                   4.300000000000000e+01     5.435000000000000e+02
                 3  'Wane'                   9.900000000000000e+01     5.435000000000000e+02
+
+JSON_ARRAYAGG
+===================================
+
+.. function:: JSON_ARRAYAGG (json_val)
+
+  Aggregate function that builds a json array out of the evaluated rows.
+
+.. code-block:: sql
+
+    CREATE TABLE t_score(name VARCHAR(10), score INT);
+    INSERT INTO t_score VALUES
+        ('Amie', 60),
+        ('Jane', 80),
+        ('Lora', 60),
+        ('James', 75),
+        ('Peter', 70),
+        ('Tom', 30),
+        ('Ralph', 99),
+        ('David', 55),
+	('Amie', 65);
+
+    SELECT JSON_ARRAYAGG (name) AS test_takers from t_score;
+::
+
+      test_takers
+    ======================
+      ["Amie","Jane","Lora","James","Peter","Tom","Ralph","David","Amie"]
+
+JSON_OBJECTAGG
+===================================
+
+.. function:: JSON_OBJECTYAGG (key_str, json_val expr)
```

Comment:
I also proposed renaming key_str as key :). This is a matter of preference, but same name should be used in both argument list and description.